MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
+MIPs are the preferred mechanism for improving Maker Governance and the Maker Protocol. MIP0 defines the *Maker Improvement Proposals Framework* for all subsequent MIPs to utilize.
+
+This foundational MIP defines the key roles of the framework and the MIP workflow; it also provides the necessary tools (templates, dependencies, processes) for working with the framework.
 
 ## Component Summary
 
-**MIP0c1: Definitions of the Maker Improvement Proposal Framework**  
-Defines several concepts that are important for understanding the MIPs process.
+- **MIP0c1: Definitions of the Maker Improvement Proposal Framework**
+    *Defines core concepts of the MIP Framework.*
+- **MIP0c2: Core Principles**  
+    *Sets the principles that all MIPs should aim to follow.*
+- **MIP0c3: The MIP Lifecycle**  
+    *Defines the formal stages in the lifecycle of MIPs from conception to approval, rejection, or deferral.*
+- **MIP0c4: MIP Components and MIP Component Types**
+    *Defines MIP Components and their types*
+- **MIP0c5: MIP Replacement Process**  
+    *Defines the process for replacing MIPs.*
+- **MIP0c6: Supporting Materials**
+    *Defines how to include external materials in MIPs.*
+- **MIP0c7: MIP Templates**
+    *Provides the MIP templates for both General and Technical MIPs.*
+- **MIP0c8: MIP0 Domain Role Dependencies**  
+    *Defines the core roles essential to the MIP Framework.*
+- **MIP0c9: Grandfathered Core Personnel Role List**  
+    *Lists the personnel currently occupying core roles.*
+- **MIP0c10: MIP Editor Role**  
+    *Defines the responsibilities of the MIP Editor role as well as the criteria and grounds for their removal.*
+- **MIP0c11: Governance Facilitator Role**  
+    *Defines the responsibilities of the Governance Facilitator role as well as the criteria and grounds for their removal.*
+- **MIP0c12: MIP Set Management**  
+    *Defines how to manage MIP Sets and the roles responsible for the task.*
 
-**MIP0c2: Core Principles**  
-Discusses some core principles that all MIPs should aim to follow.
 
-**MIP0c3: The MIP Lifecycle**  
-Lays out how a MIP is created and moves through the process to become Accepted or Rejected.
 
-**MIP0c4: MIP Components and MIP Component Types**  
-Discusses the use of components to compartmentalize and organise MIPs
+## Motivation
 
-**MIP0c5: MIP Replacement Process**  
-Discusses how MIPs can be replaced and the steps to be taken to maintain dependencies.
+MakerDAO is evolving into a trustless, fully decentralized, open-sourced, and self-sustainable organization. By offering a standardized, open-to-everyone approach to the proposal of changes to the protocol and governance, Maker Improvement Proposals (MIPs) make it possible for the Maker community to exercise long-term decentralized self-governance.
```

Comment:
This section should  be more 'why is this thing' rather than a 'what is this thing'.  So sentences should be framed in that light. 'In order to evolve into a fully decentralized and sustainable organization a standardized, open-to-everyone approach to improvements and changes is required... so that no individual or entity is able to gatekeep changes... '

Something like that.